pom.xml

```diff
@@ -67,7 +67,7 @@
         <dependency>
             <groupId>com.microsoft.azure</groupId>
             <artifactId>azure</artifactId>
-            <version>1.3.0</version>
+            <version>1.15.1</version>
```

Comment:
this will need a full test on all plugin, could you pls check with previous Jenkins people how they do validation?

Reply:
ok